uint/src/lib.rs

```diff
@@ -9,7 +9,6 @@
 //! Efficient large, fixed-size big integers and hashes.
 
 #![cfg_attr(not(feature="std"), no_std)]
-#![cfg_attr(all(not(feature="std"), test), feature(alloc))]
```

Comment:
Can you motivate this change? I.e. I'm not sure I know why it was there in the first place, nor do I know what breaks removing it, but I do know substrate is using `uint` both with and without std.

Reply:
features are enabled only on the night channel, and this feature was not used anywhere :)